src/game/profiling.c

```diff
@@ -192,15 +217,15 @@ void profiler_print_times() {
     update_total_timer();
     update_rdp_timers();
 
-#ifndef PUPPYPRINT_DEBUG
+#ifdef PUPPYPRINT_DEBUG
```

Comment:
Why was this changed? Does the standard profiler still work with `PUPPYPRINT_DEBUG` disabled?